user/clight/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "clight"
+pkgver = "4.11"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
```

Comment:
but also like half of these are actually makedepends

Reply:
I didn't grasp the difference between those yet. Hope you can explain it.